variables.tf

```diff
@@ -1,140 +1,140 @@
-# # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
 #
-#  REQUIRED
+# NETWORK
 #
-# # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-variable "name" {
+
+variable "vpc_id" {
+  description = "Required: A vpc_id in which other resources are and will be located."
   type        = string
-  description = "A plaintext name for named resources, compatible with task definition family names and cloudwatch log groups"
 }
-variable "container_definitions" {
-  #type        = any
-  description = "Container configuration in the form of a json-encoded list of maps. Required sub-fields are: 'name', 'image'; the rest will attempt to use sane defaults"
-  validation {
-    condition     = can(var.container_definitions.*.name)
-    error_message = "VALIDATION FAILURE: Every element of container_definitions must include a 'name' field."
-  }
-  validation {
-    condition     = can(var.container_definitions.*.image)
-    error_message = "VALIDATION FAILURE: Every element of container_definitions must include an 'image' field."
-  }
-  validation {
-    condition     = can(var.container_definitions[0])
-    error_message = "VALIDATION FAILURE: Variable container_definitions must be a list."
-  }
-  validation {
-    condition     = can(var.container_definitions[0])
-    error_message = "VALIDATION FAILURE: Variable container_definitions must be a list."
-  }
-  validation {
-    error_message = "VALIDATION FAILURE: Variable container_definitions.*.portMappings must all be unique."
-    condition     = length(distinct([for def in var.container_definitions : def.portMappings[0].containerPort if can(def.portMappings[0].containerPort)])) == length([for def in var.container_definitions : def.portMappings[0].containerPort if can(def.portMappings[0].containerPort)])
-  }
+
+variable "subnet_ids" {
+  description = "Required: A set of subnet_ids associated with the provided vpc_id."
+  type        = list(string)
+}
+
+variable "security_group_ids" {
+  description = "Required: A set of security_group_ids assocated with the provided vpc_id."
```

Comment:
description = "Required: A set of security_group_ids assocated with the provided vpc_id."

assocated should be associated